tests/test_app/Plugin/TestPlugin/src/Infrastructure/Table/AddressesTable.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
```

Comment:
We don't use strict typing in 3.x.  Same for the other test class added.

Reply:
Sorry, I changed target from 4.x to 3.x and forgot to remove those.